lib/barong/app.rb

```diff
@@ -57,6 +57,15 @@ def type!(key, value, options)
         when :integer
           regex!(key, value,  /^\d+$/)
           return value
+
```

Comment:
maybe we don't put an empty line?
i didn't find any tips about this style in rails.